legion/engine/llri-vk/source.cpp

```diff
@@ -7,96 +7,204 @@
 
```

Comment:
btw in this file, on my local machine, VS can't find `vulkan/vulkan.hpp` is this a file locally installed on your system? maybe put it in the deps since it doesn't seem to be a windows default included file.

Reply:
Yeah, the Vulkan SDK has to be installed on your machine https://www.lunarg.com/vulkan-sdk/